packages/disploy/src/structs/ModalActionRowComponent.ts

```diff
@@ -0,0 +1,26 @@
+import {ActionRowComponent} from "./ActionRowComponent";
+
+export class ModalActionRowComponent extends ActionRowComponent {
+    public required?: boolean;
+    public value?: string;
+    public placeholder?: string;
+
+    setRequired(required: boolean){
+        this.required = required;
+        return this;
+    }
+
+    setValue(value: string){
+        this.value = value;
+        return this;
+    }
+
+    setPlaceholder(placeholder: string){
+        this.placeholder = placeholder;
+        return this;
+    }
+
+    constructor() {
```

Comment:
Don't force users to use the builder pattern, let them pass an object to fill the class.